libs/top_level_metric_risk_levels.py

```diff
@@ -0,0 +1,97 @@
+import enum
+from typing import List
+
+from covidactnow.datapublic import common_fields
+
+
+class RiskLevel(common_fields.ValueAsStrMixin, str, enum.Enum):
```

Comment:
I don't have a huge opinion, but I wonder if it'd be better to encode the enums as an int.  I'm just thinking about when we're writing this as a timeseries, probably don't need to have the string values